From 7b7a3bde0089ddfb1cb763182fd8fd3e9b45fa31 Mon Sep 17 00:00:00 2001
From: Andrii Sultanov <andriy.sultanov@vates.tech>
Date: Fri, 11 Apr 2025 15:36:31 +0100
Subject: [PATCH] NetworkReset: Sync up the network-reset file path with xapi

It was changed in xapi back in Aug 2024
(92561391840d9de31ad70249166ee43c2de9cb8d, see
https://github.com/xapi-project/xen-api/pull/5933), without syncing up
xsconsole.

Signed-off-by: Andrii Sultanov <andriy.sultanov@vates.tech>
---
 plugins-base/XSFeatureNetworkReset.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins-base/XSFeatureNetworkReset.py b/plugins-base/XSFeatureNetworkReset.py
index c69009b..2ab99e8 100644
--- a/plugins-base/XSFeatureNetworkReset.py
+++ b/plugins-base/XSFeatureNetworkReset.py
@@ -22,7 +22,7 @@ pool_conf = '%s/pool.conf' % (Config.Inst().XCPConfigDir())
 interface_reconfigure = '%s/interface-reconfigure' % (Config.Inst().LibexecPath())
 inventory_file = '/etc/xensource-inventory'
 management_conf = '/etc/firstboot.d/data/management.conf'
-network_reset = '/tmp/network-reset'
+network_reset = '/var/tmp/network-reset'
 
 def read_dict_file(fname):
 	f = open(fname, 'r')
-- 
2.50.1

